File: clients/vscode/src/test/fixtures/csharp-small/Alpha.cs
public class Alpha {
    public int Compute(int x, int y) {
        var sum = x + y;
        var total = 0;
        for (var i = 0; i < 12; i++) {
            if (i % 2 == 0) {
                total += sum + i;
            } else {
                total += sum - i;
            }
        }
        if (total > 100) {
            total = total / 2;
        }
        return total * 2;
    }
}

File: crates/deslop/tests/fixtures/typescript-small/alpha.ts
export interface Order {
  items: Array<{ price: number; quantity: number }>;
}

export async function summarizeOrders(orders: Order[]): Promise<number[]> {
  const totals = await Promise.all(
    orders.map(async (order) => {
      const subtotal = order.items.reduce(
        (sum, item) => sum + item.price * item.quantity,
        0,
      );
      return subtotal > 100 ? subtotal * 0.9 : subtotal;
    }),
  );
  return totals.filter((total) => total > 0);
}
